src/pybind/mgr/cli_api/__init__.py

```diff
@@ -1,6 +1,5 @@
 import os
 from .module import CLI  # noqa # pylint: disable=unused-import
```

Comment:
In order to run the unit tests properly, this import should go after the conditional `import tests`.